packages/compartment-mapper/src/node-powers.js

```diff
@@ -61,18 +88,18 @@ const makeReadPowersSloppy = ({ fs, url = undefined, crypto = undefined }) => {
     });
     await promise;
 
-    const path = fileURLToPath(location);
+    const filepath = fileURLToPath(location);
```

Comment:
Consider `filePath` or merely `path`, since `path` is not otherwise in scope and `location` has no qualifier like `fileLocation`.

Reply:
`path` is in scope